app/javascript/modules/listingDetailsAside/ListingDetailsProcess.tsx

```diff
@@ -9,11 +10,13 @@ export interface ListingDetailsProcessProps {
 }
 
 export const ListingDetailsProcess = ({ listing }: ListingDetailsProcessProps) => {
+  const isApplicationOpen = isOpen(listing)
   return (
     <>
       {!!listing.Lottery_Date &&
         dayjs(listing.Lottery_Date) > dayjs() &&
-        !listing.LotteryResultsURL && (
+        !listing.LotteryResultsURL &&
+        isApplicationOpen && (
```

Comment:
i see we're calling `isApplicationOpen ` one parent above in `ListingDetailsAside`, we could consider moving the call up to avoid making an additional call here